frontend/src/components/HandbookOverlay/HandbookAttacks.tsx

```diff
@@ -22,10 +22,10 @@ function HandbookAttacks({ currentLevel }: { currentLevel: LEVEL_NAMES }) {
 
 	return (
 		<article className="handbook-page">
-			<header>
```

Comment:
Interested to know why you changed this. MDN seems to indicate `<header>` is legitimate here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header

Reply:
During development I was trying to avoid confusing myself with the headers since the OverlayHeader is a header, but you're right, I'll change it back to a header. 
It also adds a nice visual presentation:
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/d78aa425-808c-43b2-ad35-c8559e264943)

Reply:
yep, will change it back to header and use the class selector instead for the styling of the first header